conans/test/utils/file_server.py

```diff
@@ -58,6 +58,7 @@ def get_user_passwd(file):
                 return bottle.HTTPError(401, "Not authorized")
             return bottle.static_file(file, store)
 
+        @app.route("/internal_error/<rest>", method=["GET"])
```

Comment:
Added to ensure that extra args also trigger this, seems not to break unralted tests